x/evm/keeper/abci.go

```diff
@@ -22,11 +22,12 @@ func (k *Keeper) BeginBlock(ctx sdk.Context, req abci.RequestBeginBlock) {
 	ctx = ctx.WithGasMeter(sdk.NewInfiniteGasMeter())
 
 	// Set the hash -> height and height -> hash mapping.
-	hash := req.Header.LastBlockId.GetHash()
-	height := req.Header.GetHeight() - 1
```

Comment:
why do we not "- 1" the blockheight anymore? has the GetHeight() method changed?

Reply:
1. The code before modification thinks:
In the beginBlock stage of "height n", only the block hash of "height n-1" can be obtained, but the block hash of "height n" cannot be obtained.

2. Now we find:
In the beginBlock stage of "height n", not only the block hash of "height n-1" can be obtained, but also the block hash of "height n" can be obtained.

3. Therefore:
We don’t need to record the block hash of “height n” when it is “height n+1”, we can record the block hash of “height n” when it is “height n”

Reply:
GetHeight() has not changed